jsonobject/jsonobject.go

```diff
@@ -1,4 +1,4 @@
-package dl
+package jsonobject
```

Comment:
When json_object.go was in the root folder, we can assign it with any package name we want. (it is out of the Go source code structural package rule)

After this import statement (tell the importer to search the root folder)
```
import "github.com/scalar-labs/scalardl-go-client-sdk/v3"
```

it doesn't matter what your package name is, the importer can find it.

For example, if you have a Go file in the root folder defined with package foo.

```
package foo

struct Foo {
}
```

then, you can import it with
```
import "github.com/scalar-labs/scalardl-go-client-sdk/v3"

foo.Foo{
}
```

if your change the package name to
```
package bar

struct Foo {
}
```

you can import it with
```
import "github.com/scalar-labs/scalardl-go-client-sdk/v3"

bar.Foo{
}
```

This `feature` makes things a bit unclear than just
```
import "github.com/scalar-labs/scalardl-go-client-sdk/v3/foo"

foo.Foo{
}
```

This change is to make it more clear.